onnxruntime/core/providers/cpu/controlflow/loop.cc

```diff
@@ -203,12 +207,18 @@ Status LoopImpl::Initialize() {
                            " but has ", num_subgraph_outputs);
   }
 
+  if (init_condition_rank_ >= 2 || init_iter_num_rank_ >= 2) {
+    return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL,
+                           "'Loop' input 0 and 1 should be a scalar tensor, but has rank ",
```

Comment:
> 0 and 1 [](start = 40, length = 8)

Clearer to use the input names instead of positions. 
'but have ranks of '

Reply:
done